app/players/player.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    team = []

```

Comment:
The `team` variable is a class variable, which means it's shared across all instances of the class. If you create multiple `Player` instances, they will all share the same `team` list, which might not be what you want. If each `Player` is supposed to have its own `team`, you should make `team` an instance variable by defining it inside the `__init__` method.